frontend/src/test/scala/bloop/tasks/IntegrationTestSuite.scala

```diff
@@ -1,41 +1,43 @@
 package bloop.tasks
 
 import java.nio.file.{Files, Path, Paths}
+import java.util.{Arrays, Collection}
+
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.experimental.categories.Category
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameters
 
 import bloop.cli.{Commands, ExitStatus}
 import bloop.engine.{Dag, Exit, Interpreter, Run}
 import bloop.exec.JavaEnv
-import bloop.{DynTest, Project}
+import bloop.Project
 import bloop.io.AbsolutePath
 
-object IntegrationTestSuite extends DynTest {
-  val projects = Files.list(getClass.getClassLoader.getResources("projects") match {
-    case res if res.hasMoreElements => Paths.get(res.nextElement.getFile)
-    case _ => throw new Exception("No projects to test?")
-  })
+object IntegrationTestSuite {
+  val projects = Files
+    .list(getClass.getClassLoader.getResources("projects") match {
```

Comment:
Can we not reuse `loadTestProject` here? I think this logic could be moved somewhere better than the test case.

Reply:
Yup, this has improved in #156 :)